plugins/hydra_ax_sweeper/hydra_plugins/hydra_ax_sweeper/ax_sweeper.py

```diff
@@ -112,7 +112,7 @@ def setup(
             config=config, config_loader=config_loader, task_function=task_function
         )
 
-    def sweep(self, arguments: List[str]) -> None:
+    def sweep(self, arguments: List[str]) -> Any:
```

Comment:
what is is the type of the result from Ax?

Reply:
`Optional[Tuple[Dict[str, Union[str, float, int, None]], Optional[Tuple[Dict[str, float], Optional[Dict[str, Dict[str, float]]]]]]]`

From: https://ax.dev/api/service.html#ax.service.ax_client.AxClient.get_best_parameters